retailer-oas/retailer.yaml

```diff
@@ -326,14 +471,12 @@ paths:
               schema:
                 $ref: '#/components/schemas/DeliveryArea'
 
-  /delivery-areas/{id}/slot-availability:
+  /delivery-area/{id}/delivery-timeslots:
```

Comment:
We should review this together. I agree with the specification of "delivery timeslots" and that they should be the ones included in the MVP. In the future we might want to have a `/picking-timeslots` and a `/timeslots` that encompasses all the slot types.